Navigation.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+import { NavigationContainer } from "@react-navigation/native";
+
+import HomeScreen from "./screens/Home";
+import AboutScreen from "./screens/About";
+import HistoryScreen from "./screens/History";
+import CategoryScreen from "./screens/Category";
+
+export default function Navigation() {
+  const Stack = createStackNavigator();
+
+  return (
+    <NavigationContainer>
+      <Stack.Navigator headerMode={"none"} initialRouteName={"Home"}>
```

Comment:
```suggestion
      <Stack.Navigator headerMode="none" initialRouteName="Home">
```
Curly braces are not necessary for string props 😛

Reply:
This applies to any props wrapped like that actually :suspect: